<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:  #19822d;
background-color: #1a1d1b;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000;
border-radius: 5px;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_graphics_view_button">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>440</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  #19822d;
border: 2px solid #19822d;
font-size: 20px;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="make_a_guess_button">
    <property name="geometry">
     <rect>
      <x>589</x>
      <y>490</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  #19822d;
border: 2px solid #19822d;
font-size: 20px;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Получить ответ</string>
    </property>
   </widget>
   <widget class="QLabel" name="answer">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3d3d3d;
border-radius: 5px;
font-size: 300px;</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="margin">
     <number>20</number>
    </property>
   </widget>
   <widget class="QPushButton" name="load_network_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  #19822d;
border: 2px solid #19822d;
font-size: 20px;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Загрузить сеть</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="hidden_layers_quantity">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество скрытых слоев</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nq_on_first">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nq_on_second">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nq_on_third">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nq_on_fourth">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nq_on_fifth">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="create_network_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  #19822d;
border: 2px solid #19822d;
font-size: 20px;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Создать сеть</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="matrix_network">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Матричная</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="graph_network">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Графовая</string>
    </property>
   </widget>
   <widget class="QPushButton" name="train_net">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  #19822d;
border: 2px solid #19822d;
font-size: 20px;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Обучить сеть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_bmp">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>440</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  #19822d;
border: 2px solid #19822d;
font-size: 20px;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Загрузить BMP</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="epoch_quantity">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество эпох обучения</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="part_of_test">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Часть тестовой выборки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="test_net">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  #19822d;
border: 2px solid #19822d;
font-size: 20px;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Протестировать сеть</string>
    </property>
   </widget>
   <widget class="QLabel" name="answer_probability">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>440</y>
      <width>400</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3d3d3d;
border-radius: 5px;
font-size: 30px;</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="margin">
     <number>20</number>
    </property>
   </widget>
   <widget class="QPushButton" name="save_net">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  #19822d;
border: 2px solid #19822d;
font-size: 20px;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Сохранить сеть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>925</x>
      <y>40</y>
      <width>290</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер распознанного класса (нумерация с 1)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество распознаваемых классов</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="output_classes_quantity">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>26</number>
    </property>
   </widget>
   <widget class="QPushButton" name="train_on_img">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>490</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  #19822d;
border: 2px solid #19822d;
font-size: 20px;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Обучить на картинке</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>540</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер правильного класса</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="right_class_number">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>540</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
